### ISSUE TITLE
Listbox: Not possible to select Listbox options using iPhone with VoiceOver on.

### ISSUE THREAD

**Issue by aappoalander** [reach/reach-ui#625]
### Current Behavior

Currently it seems to be impossible to select Listbox options using an iPhone (multiple different models) with Safari or Chrome when VoiceOver is on. iPad with VoiceOver on and at least some Android devices seem to be working without issues.

The options can be focused with single tap, but double tapping seems to have no effect. Selecting options does not seem to be possible with other gestures either.

### Expected behavior

Listbox options should be selectable with double tap using iPhone with VoiceOver on.

### Reproducible example

https://reacttraining.com/reach-ui/listbox

### Suggested solution(s)

No idea how to fix this. Some other select-component type library implementations seem to be working, e.g. https://material-ui.com/components/selects/

### Your environment

Multiple iPhones using Chrome and Safari and listbox documentation examples. VoiceOver on.

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Reach Package    |     Listbox    |    0.10.4?     |
| Browser          |         |     Chrome, Safari    |
| Assistive tech   |         |    VoiceOver     |
| Operating System |         |    iOS (only iPhone)    |

**Comment by chancestrickland** [comment]
Very odd. Confirmed the bug on my end but I'm not sure immediately what the issue is. Tried using click and touchend events to select options just to be sure, no dice there. Will keep investigating this, thanks!

**Comment by chaance** [comment]
This should be addressed in the latest release. Please let me know if you still have issues after upgrading and I can re-open if needed.

**Comment by mezzab** [comment]
@chaance 

I'm having an issue when selecting a menu item with VoiceOver, and I think this issue might be related. 

The problem is that in order to select a MenuButton it requires two selections.
The issue is also reproducible with Talkback in Android.

Leaving a video of the issue: 
https://user-images.githubusercontent.com/7173948/121224207-b55a3100-c85e-11eb-8840-c547f94b6663.mp4
